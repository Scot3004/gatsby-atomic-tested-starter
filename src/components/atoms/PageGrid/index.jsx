import styled from 'styled-components';

const PageGrid = styled.div`
  display: grid;
  grid-template-columns: 1fr;
`;

export default PageGrid;
