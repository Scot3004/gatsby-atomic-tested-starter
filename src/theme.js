const theme = {
  backgroundColor: '#f5f5f5',
  textColor: '#000',
  mutedColor: '#555',
  primaryColor: '#663399',
  visitedColor: '#402060'
};

export default theme;
