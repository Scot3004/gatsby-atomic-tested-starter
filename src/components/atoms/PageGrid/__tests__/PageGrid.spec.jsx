import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import theme from '../../../../theme';
import PageGrid from '..';

describe('When I visit the site', () => {
  const tree = renderer.create(<PageGrid theme={theme} />).toJSON();
  test('Then a grid for the page should be shown', () => {
    expect(tree).toMatchSnapshot();
  });
});
