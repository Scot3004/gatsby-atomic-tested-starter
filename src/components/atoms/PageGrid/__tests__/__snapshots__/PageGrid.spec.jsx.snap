// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When I visit the site Then a grid for the page should be shown 1`] = `
.c0 {
  display: grid;
  grid-template-columns: 1fr;
  grid-template-areas: 'navigation' 'sidebar' 'main';
}

@media only screen and (min-width:48em) {
  .c0 {
    grid-template-columns: 1fr 3fr;
    grid-template-areas: 'sidebar navigation' 'sidebar main';
    grid-gap: 2rem;
    grid-row-gap: 0.5em;
  }
}

<div
  className="c0"
/>
`;
